apps/estuary/mix.exs

```diff
@@ -32,8 +32,6 @@ defmodule Estuary.MixProject do
       {:credo, "~> 1.0", only: [:dev], runtime: false},
       {:dead_letter, in_umbrella: true},
       {:distillery, "~> 2.1"},
-      {:divo, "~> 1.1", only: [:dev, :integration]},
-      {:divo_kafka, "~> 0.1.5", only: [:dev, :integration]},
```

Comment:
I'm assuming this is related to the removal of the step in the workflow for estuary? (same with flair divo dep removals too)? 

Are there no integration tests for estuary and flair, and divo was an unused dependency / unused gh action step?

Reply:
Yep, some run steps were removed for apps without integration tests, and divo was removed for apps that don't use it.